### PR TITLE
feat: add stabilization of requests and istio monitoring addons with …

### PR DESCRIPTION
**Pull request by JanainaMK**
- Requests for continuous experimentation are now stabilized. Model-service rollout version is only selected randomly the first time and all other times the same version is used. More details can be read in the README.

- Removed installation of Prometheus with helm. Prometheus and Grafana are now installed as Istio addons along with other tools such as kiali and jaeger. Addon files were taken from the provided Istio samples [here](https://github.com/istio/istio/tree/master/samples/addons). 

- Alerts are now included and can be seen in the Prometheus dashboard. Unfortunately, an Alertmanager that handles alert notifications could not be configured.

TO-DO:
- Update tags for model-service to versions where the correct models are used for the continuous experimentation.

### CODE REVIEW

File: charts/application/custom-dashboards/metrics-vis.json
Comment (alemoraru) [inline charts/application/custom-dashboards/metrics-vis.json:1]: Should we not try at all to use the custom dashboards as a file? Adding directly in a file seems rather tedious, no?

**Review by alemoraru — COMMENTED** [review]
Just minor general comments (that I've also left on the PR itself):
- I personally think we should still go for the remote helm chart storage. Currently, this does not work out of box. Let's address this in a follow-up PR so reviewers (including TAs) will easily review this without having to enable everything. If this is a simple `helm install <chart-name>`, then it will surely aid us in a better grade and product.

Other than this, good work on the clean-up and implementation! I was wondering whether to leave stand-alone k8s files, but that's overkill to maintain.

File: charts/application/custom-dashboards/metrics-vis.json
Comment (JanainaMK) [reply charts/application/custom-dashboards/metrics-vis.json:1]: I agree but I didn't find a way to get an external file to work for the ConfigMap and appear properly in the Grafana dashboard. If you know a way to get it to work, feel free to change it.

File: charts/application/Chart.yaml [hunk 1]
```diff
@@ -13,9 +13,3 @@ version: 0.1.20
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
 appVersion: "0.1.20"
-
-#dependencies:
```
[the remark this answers is not in the document]
Comment (JanainaMK) [reply charts/application/Chart.yaml:17]: Yea Prometheus can work without the other tools. However, I haven't found a way to incorporate them as one helm chart installation that works with Istio, so the installation is separate as you see here. This is also the installation that was suggested in the assignment material under continuous experimentation, where the monitoring is handled by Istio. I'm not sure how much they want us to stick to this though. I'm open to any other changes but this what I found currently works and have seen other groups do as well.

**Review by alemoraru — APPROVED** [review]

**Pull request merged**